mono/profiler/proflog.c

```diff
@@ -232,19 +232,20 @@ typedef struct _LogBuffer LogBuffer;
  * [pointer: sleb128] pointer of the metadata type depending on mtype
  * if mtype == TYPE_CLASS
  *	[image: sleb128] MonoImage* as a pointer difference from ptr_base
- *  [flags: uleb128] must be 0
+ *	[flags: uleb128] must be 0
  * 	[name: string] full class name
  * if mtype == TYPE_IMAGE
- *  [flags: uleb128] must be 0
+ *	[flags: uleb128] must be 0
```

Comment:
The change is supposed to fix indentation, as the lines below are using tabs, not spaces, as the lines being changed do